scripts/nodelib.py

```diff
@@ -99,7 +99,7 @@ def run_docker(image, home_dir, boot_nodes, verbose):
     if verbose:
         envs.extend(['-e', 'VERBOSE=1'])
     subprocess.check_output(['docker', 'run',
-                    '-d', '--network=host', '-v', '%s:/srv/near' % home_dir,
+                    '-d', '-p', '3030:3030', '-v', '%s:/srv/near' % home_dir,
```

Comment:
this is not enough, because the port might be different.
To do it properly, we should read the port from the `home_dir + '/config.json'`:

For example:
```
>>> import json
>>> config = json.loads(open('/Users/cypress/.near/config.json').read())
>>> config['rpc']['addr'][config['rpc']['addr'].find(':') + 1:]
u'3030'
>>> config['network']['addr'][config['network']['addr'].find(':') + 1:]
u'24567'
```

Reply:
fixed